resources/devices/mobile-phone/htc.json

```diff
@@ -922,19 +922,6 @@
             "isTablet": "false"
         }
     },
-    "HTC Elf": {
```

Comment:
A lot of the devices here seemed to be duplicates of each other (one with the "common" name, and another with the "code" name), so I tried to combine them wherever possible.